### PR TITLE
GVF message updated with a new field

### PR DESCRIPTION
**Pull request by noether**
New field for the update of the GVF in Paparazzi

### CODE REVIEW

**Comment by flixr** [comment]
plz fix the indentation...

**Comment by noether** [comment]
yeah, I will :P

**Comment by noether** [comment]
oki, done. This pull request has to be done at the same time as this other one: https://github.com/paparazzi/paparazzi/pull/2052

thanks!

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -536,12 +536,13 @@
...
-    <description>
-    Information about the trajectory followed by the Guidance Vector Field algorithm.
-    </description>
+      <description>
+      Information about the trajectory followed by the Guidance Vector Field algorithm.
+      </description>
       <field name="error" type="float">Error index e, i.e. 'distance' to the trajectory</field>
       <field name="traj" type="uint8">Kind of trajectory</field>
       <field name="s" type="int8" values="1|-1">Direction to be followed</field>
+	  <field name="ke" type="float">Gain for the vector field convergence</field>
```
Comment (podhrmic) [inline message_definitions/v1.0/messages.xml]: Pls fix indentation here @noether

**Pull request merged**